internal/pkg/config/testdata/test-config.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+# Use mixed lower and upper case keys here to test robustness of YAML package.
```

Comment:
If the YAML package allows it, that's fine, but why do we need to test it?

Reply:
I was of two minds on how to document this.  I (anecdotally) feel like users will tend towards lower-cased keys when writing a YAML.  That's been my experience in what I've seen, anyway.

Part of this is from a pain point of trying to use the `gopkg.in/yaml` package at first, which can be very finicky about key casing.  But without that context, I can see how it would be confusing.

I'll drop the mixed-cases for the sake of clarity and cleanliness.

Reply:
Sounds good.

My personal point of view is that any deviation in casing should not be allowed, but it's not a big deal if the YAML package takes care of that for us.

Reply:
Yeah, good top-level-comment that we are essentially testing the imported package rather than our own.  Cleaned up.